README.md

```diff
@@ -0,0 +1,64 @@
+# FanNetworking
+
+This base layer delivers the "Ubuntu fan" that is an extension of the network
+tunnel driver in the Linux kernel.
+
+Containers enable dense virtualization, it is easy to run hundreds or thousands
+of containers on a single host machine. All of these containers need an IP
+address and one solution is to use the fan.
+
+The fan is a mapping between a smaller network address space (typically a /16
+network) and a larger one (typically a /8), which assigns subnets from the
+larger one to IP addresses on the smaller one, and enables automatic and simple
+tunnelling and routing between systems on the larger address space.
+
+The fan trades access to one user-selected /8 (potentially external) address
+range for an expanded pool of addresses to be used by containers or virutal
+machines. Fan does this by mapping the addresses in a way that can be computed,
+rather than one that requires maintenance of distributed state.
+
+For more information go to the [Ubuntu wiki page about Fan
+networking](https://wiki.ubuntu.com/FanNetworking)
+
+# Usage
```

Comment:
I think you should show how to override the "profile" declared in the `defines:` bit of the layer.yaml def. when using this